### ISSUE TITLE
Is this now the official repository?

### ISSUE THREAD

**Issue by rgl** [jsslkeylog/jsslkeylog#2]
Is this now the official repository?

If it is, it would be nice to have @schierlm as a member of the https://github.com/jsslkeylog organization and some blurb in the README :-)

@schierlm do you confirm this is now the official repository?

**Comment by schierlm** [comment]
Hello Rui,

yes this is the offical repo. I am not only a member, but admin of the organization you mentioned. Is this not displayed publicly? (I have disabled the display of my project memberships on my profile page, and I'd like to keep that disabled. If there is a way to show my membership in the organization if you discovered it (but not the other way round), I can surely change this)

Feel free to open a pull request for the README to make this clearer.

(I also set the project new address on SourceForge and made the old website redirect to the new website here on GitHub as well. This should be things only possible for the old owner).

My near future plans are testing jsslkeylog against the new TLS backport in Java 8 (especially TLS 1.3), but I don't find time. If I do, I'll release a new version and try to polish the website and stuff. My hope was that when I move to GitHub, others might help me in doing so, but so far I am not very happy about it (the guy who moved the website for me also wanted to move the old releases, but so far he has not uploaded the artifacts to the newly created tags).
